### DISCUSSION TITLE
没有函数savehtml

### DISCUSSION THREAD

**Discussion by Zwei1996** [linuxdeepin/developer-center#3393] [Q&A | 问答板块] [answered]
您好，在docparser项目中，里面每个文件转换下的readme里有提savehtml这个函数，但是实际上并没有，请问这个函数有相应的源码吗，谢谢

**Comment by BLumia** [comment, marked as answer]
看上去相关代码是基于 https://github.com/dmryutov/document2html/ 这个项目的，savehtml 位于 [fileext.hpp](https://github.com/dmryutov/document2html/blob/753d37a42a7c9e9daeb33183228ac5bf0df87295/src/libs/fileext/fileext.hpp#L68)。可能 docparser 附带的是修改过的版本，去掉了这个函数吧。

**Reply by Zwei1996** [reply]
找到了，谢谢您
